### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by PrinceBadru**
## Description.
In this project you will add some new tables and add foreign key columns to your existing animals table to model one-to-many relationships. Afterward, you will use JOIN to query the data.
 - Create a table named owners with columns of relative data.
 - Create a table named species with columns of relative data.
 - Write queries (using JOIN) to answer the following questions:

1. What animals belong to Melody Pond?
2. List of all animals that are pokemon (their type is Pokemon).
3. List all owners and their animals, remember to include those that don't own any animal.
4. How many animals are there per species?
5. List all Digimon owned by Jennifer Orwell.
6. List all animals owned by Dean Winchester that haven't tried to escape.
7. Who owns the most animals?

 - Remember all these should be written in queries.sql file.
 - Screenshot of the query that shows the required changes.
1. What animals belong to Melody Pond?
Screenshot (55)

2. List of all animals that are pokemon (their type is  @Pokemon).
Screenshot (56)

3. List all owners and their animals, remember to include those that don't own any animal.
Screenshot (57)

4. How many animals are there per species?
Screenshot (58)

5. List all Digimon owned by Jennifer Orwell.
Screenshot (59)

6. List all animals owned by Dean Winchester that haven't tried to escape.
Screenshot (60)

 -   (-- 7. Who owns the most animals?)
Screenshot (53)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+-- Modify the animals table
 ALTER TABLE animals
-ADD COLUMN species VARCHAR(255);
+ADD COLUMN species_id INTEGER REFERENCES species(id),
+ADD COLUMN owner_id INTEGER REFERENCES owners(id);
```
Comment (Abenezer-Tilahun) [inline schema.sql]: - [ ] As stated in the project requirement Kindly make sure that Modify animals table by Remove column species So kindly write the Sql for Removing column species just like the below sample. 

   ```
ALTER TABLE animals 
  DROP column species;
  ADD COLUMN species_id INTEGER REFERENCES species(id),
 ADD COLUMN owner_id INTEGER REFERENCES owners(id);
```

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @PrinceBadru 

Great job for implemented  the some of the Project Requirement  , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ DB Schema is properly updated with new tables
✔️ All data is correctly added 
✔️ Great query organization
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages.
✔️ Good job for Implemented Vet clinic database: query multiple tables.

almost

## Required Changes ♻️
_Check the comments below the review._

- [x] Kindly make sure that You  add a all screenshots of yours queries results to the pull request description.

- [x] Kindly make sure that your README should have  the section **Future Features, including 1-3 potential features to be added.**

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @PrinceBadru👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- Good job with multiple tables. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️

- [ ] As requested by the previous reviewer, Please make sure to add screenshots of results of all the queries to the pull request description. Currently, I can see only one query result screenshot.
- [x] As requested by the previous reviewer, Please make sure the README file should have the section Future Features, including 1-3 potential features to be added.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrinceBadru** [comment]
> ## STATUS: CHANGES REQUIRED ♻️
> ## Hi @PrinceBadru👋,
> Great work on making the changes requested by a previous reviewer 👏 You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> 
> ## To Highlight 🎉
> * Good job with multiple tables. ✔️
> * Gitflow is followed. ✔️
> * Professional README file. ✔️
> * PR has a good title and summary. ✔️
> 
>  [ 68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966 ](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) [ ](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)
> 
> ## Required Changes ♻️
> * [ ]  As requested by the previous reviewer, Please make sure to add screenshots of results of all the queries to the pull request description. Currently, I can see only one query result screenshot.
> * [ ]  As requested by the previous reviewer, Please make sure the README file should have the section Future Features, including 1-3 potential features to be added.
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@thecodechaser & @Abenezer-Tilahun , I have this instruction " Take a screenshot of the results of your queries. " , when I put run/ execute all of my query.sql code I get that response  here is an extended view of my screenshot that has all the extensive code 
Screenshot (54)
I donot know what more you want , t this point I am confused.

**Review by topeogunleye — APPROVED** [review]
### Hi @PrinceBadru, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Added screenshots of results of all the queries to the pull request description.:heavy_check_mark:
- Added three future features to the README file. ✔️ 
 
## [Optional] suggestions:
👉 One optional suggestion for improvement is to follow the convention for naming your branch. Usually, branch names should use the past tense verb to indicate that the feature has been implemented. For example, instead of `query-multiple-tables`, you could use `queried-multiple-tables`. This would make your branch name more consistent and clear for future pull requests.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**